master/pkg/aproto/exit.go

```diff
@@ -73,6 +73,9 @@ const (
 	// ContainerAborted denotes the container was canceled before it was started.
 	ContainerAborted = FailureType("container was aborted before it started")
 
+	// ContainerMissing denotes the container was missing when the master asked about it.
+	ContainerMissing = FailureType("request for action on unknown container")
```

Comment:
do you want to change it 
```suggestion
	ContainerMissing FailureType = "request for action on unknown container"
```

in this file as well?